index.css

```diff
@@ -0,0 +1,1843 @@
+  }
+
+  .headline-section {
+    width: 327px;
+    height: 192px;
+  }
+
+  .headline-heading {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    text-align: center;
+    color: #172b4d;
+    margin: 0;
+  }
+
+  .headline-heading span {
+    color: rgb(92, 129, 92);
+  }
+
+  .supporting-text {
+    width: 327px;
+    height: 164px;
+  }
+
+  .text {
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 28px;
+    text-align: center;
+    color: #172b4d;
+  }
+
+  .social-media-icons {
+    width: 170.17px;
+    height: 20px;
+    padding-top: 30px;
+    display: flex;
+    flex-direction: row;
+    justify-content: baseline;
+    align-items: center;
+    gap: 20px;
+    padding-left: 0;
+    margin: 0;
+  }
+
+  .social-media-icons > li {
+    list-style: none;
+  }
+
+  .normal-button {
+    width: 327px;
+    height: 48px;
+    display: flex;
+    flex-direction: row;
+    justify-content: center;
+    align-items: flex-start;
+  }
+
+  .btn {
+    width: 40px;
+    height: 40px;
+    background: #fff;
+    padding: 8px;
+    border: 1px solid #dfe1e6;
+    border-radius: 32px;
+  }
+
+  .btn:disabled {
+    cursor: none;
+  }
+
+  .disabled-button {
+    width: 24px;
+    height: 24px;
+  }
+
+  .work {
+    height: 3283px;
+    gap: 58px;
+    width: 379px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+  }
+
+  .work-heading {
+    width: 327px;
+    height: 125px;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    padding: 0;
+    gap: 24px;
+  }
+
+  .recent-works {
+    width: 327px;
+    height: 97px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    font-weight: 700;
+    font-size: 40px;
+    line-height: 52px;
+    text-align: center;
+    color: #172b4d;
+  }
+
+  .indicator_horizontal {
+    width: 48px;
+    height: 4px;
+    border-radius: 24px;
+  }
+
+  .main-content-cards {
+    width: 331px;
+    height: 2972px;
+    display: grid;
+    grid-row: 1 / 7;
+    row-gap: 48px;
+    padding: 4px 2px;
+    align-items: center;
+  }
+
+  .card-work-1 {
+    width: 327px;
+    height: 474px;
+    display: flex;
+    flex-direction: column;
+    background: #ebf0ee;
+    border: 1px solid #d0d9d4;
+    border-radius: 8px;
+  }
+
+  .image {
+    width: 329px;
+    height: 252px;
+  }
+
+  .content {
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    gap: 16px;
+    width: 327px;
+    height: 474px;
+    background: #fff;
+    border-radius: 8px;
+  }
+
+  .header-work-1 {
+    width: 295px;
+    height: 57px;
+    font-family: 'Inter', sans-serif;
```

Comment:
- [ ] It is noticed that ` font-family: 'Inter', sans-serif;` property is repeated multiple times, Kindly let us stick to the DRY principle by not repeating a property more than it should 🙂. This can be done by having a class called `.pry-font`  and assigning it wherever possible as a class name in your `html` markup code.
```
.pry-font {
  font-family: 'Inter', sans-serif;
}

.flex-column {
  display: flex;
  flex-direction: column;
}
```

```
<ul class="pry-font ">
    <li>
        <a href="https://twitter.com/AleMdta"><img src="./images/twitter.svg" alt=""></a>
    </li>
  <li>
      <a href="#"><img src="./images/medium.svg" alt=""></a>
  </li>
  <li>
      <a href="https://github.com/AleMdta"><img src="./images/github.svg" alt=""></a>
  </li>
    <li>
        <a href="#"><img src="./images/angellist.svg" alt=""></a>
    </li>
</ul>
```
OR having it as the font family for the body element, while changing the font-family in a different class section.
- And kindly attend to any repetitions in your code 🙂.
- Kindly note that DRY is a requirement, but since the other reviewers have been overlooking it, I've decided to mark it as optional. 😉